src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -25,6 +25,9 @@ public string Domain
         [SupportedOSPlatform("windows")]
         public bool LoadUserProfile { get; set; }
 
+        [SupportedOSPlatform("windows")]
+        public bool UseCredentialsForNetworkingOnly { get; set; }
```

Comment:
since this is a new public API, it needs and XML doc

Reply:
ACK. I wasn't sure wheter it was okay to simply add them as `///` blocks or if I should add them to `dotnet/dotnet-api-docs` in a separate PR, but now found https://github.com/dotnet/runtime/blob/32ac664fca93cb692597e75021ae5e8ee7b75b1c/docs/coding-guidelines/adding-api-guidelines.md, which answers that question.